src/GrassPokemon.java

```diff
@@ -0,0 +1,7 @@
+public class GrassPokemon extends Pokemon {
+
```

Comment:
Er ontbreken hier nog klasse variabelen en methoden zoals beschreven in de randvoorwaarden van de opdracht